tests/python/rest_api/test_jobs.py

```diff
@@ -724,11 +742,11 @@ def test_can_get_gt_job_chunk(self, admin_user, tasks, task_mode, quality):
                     assert image.size > (1, 1)
                     assert np.any(image_data != 0)
 
-    def _get_or_create_gt_job(self, user, task_id, frames):
+    def _create_gt_job(self, user, task_id, frames):
         with make_api_client(user) as api_client:
             (task_jobs, _) = api_client.jobs_api.list(task_id=task_id, type="ground_truth")
             if task_jobs.results:
-                gt_job = task_jobs.results[0]
+                raise RuntimeError(f"gt_job already exist for task {task_id}")
```

Comment:
There is no need to check this additionally, the job creation request will fail with an error if there is a gt job already.